Malay-Kumar-Parida/02.Variables/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import "fmt"
+
+var l float32 = 100 //lower case Thorughout This package
+var L int = 50      //Visible everywhere
+//Cant used 3rd type of declarration here
```

Comment:
Always give a space after // 
Suggested change: `// Cant used 3rd type of declarration here`